package.json

```diff
@@ -10,7 +10,8 @@
     "eslint-config-prettier": "9.1.0",
     "prettier": "3.2.5",
     "typescript": "5.5.4",
-    "typescript-eslint": "7.6.0"
+    "typescript-eslint": "7.6.0",
```

Comment:
could be worth also adding `"reflect-metadata": "^0.2.2"` for us folks without it already installed on our computers 😄

Reply:
you know - I did add it at first, but then I saw that it was in base, so I thought maybe it would pick it up, and when I ran `npm run fresh` and `npm run install-all` it worked, so I thought it would be okay without it, but I suppose it is better safe the sorry! I added it 👍